sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/CommentTodo.vb

```diff
@@ -0,0 +1,16 @@
+﻿Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+Imports System.Text
+
+Namespace Tests.TestCases
+    ''' <summary>
+    ''' TODO: Do something ' Noncompliant
```

Comment:
I cannot test for the issue message because it contains the word TODO in a comment and so triggers another issue on the same line which looks really odd when reviewing.